pmm-app/src/pmm-dbaas/components/DBCluster/PSMDB.service.ts

```diff
@@ -21,7 +21,7 @@ const DBCLUSTER_STATUS_MAP = {
 };
 
 export class PSMDBService extends DBClusterService {
-  getDBClusters(kubernetes: Kubernetes): Promise<DBClusterPayload> {
+  async getDBClusters(kubernetes: Kubernetes): Promise<DBClusterPayload> {
```

Comment:
If really needs async here change in the parent too.

Reply:
no need, it was added for testing purpose, removed